app/models/game.rb

```diff
@@ -1,4 +1,5 @@
 class Game < ApplicationRecord
+  scope :available, -> { where(available: true) }
```

Comment:
For this scope, you'll want the query to return games where `white_player_id` or `black_player_id` is null. (there is an empty spot).

So something like this `where('black_player_id IS NULL or white_player_id IS NULL')`